duqtools/__main__.py

```diff
@@ -72,10 +73,23 @@ def cmdline():
                                default=False,
                                help='Fancy progress bar')
 
+    parser_init = subparsers.add_parser('init',
+                                        help='Create a default config file',
+                                        parents=[parser],
+                                        conflict_handler='resolve')
+    parser_init.set_defaults(func=init)
+
```

Comment:
This parser already exists? it seems duplicated

Reply:
Yep, copy/paste error, I removed it 👍